### DISCUSSION TITLE
bot.waitForTicks

### DISCUSSION THREAD

**Discussion by David-Launikitis** [PrismarineJS/mineflayer#1682] [Q&A]
Can someone write out a simple example using bot.waitForTicks()? I am struggling to use this function. Thanks!

**Comment by amoraschi** [comment]
1 tick = 0.05 seconds

1 second = 20 ticks

```js
await bot.waitForTicks(20)
```

**Reply by David-Launikitis** [reply]
`async function spawnTimer() {
    await bot.waitForTicks(100)
    console.log("new spawn ^")
}`

` UnhandledPromiseRejectionWarning: TypeError: bot.waitForTicks is not a function`

I am using mineflayer 3.0.0

**Reply by amoraschi** [reply]
Put it inside the spawn event
